Dockerfile

```
@@ -1,25 +1,18 @@
-FROM golang:1.10.1-alpine3.7 as helmfile
-ENV HELMFILE_VERSION v0.45.3
-RUN apk add --no-cache make git
-RUN go get github.com/roboll/helmfile
-WORKDIR /go/src/github.com/roboll/helmfile/
-RUN git checkout ${HELMFILE_VERSION}
-RUN make static-linux
+FROM alpine:3.9.3
 
-
-FROM alpine:3.8
-
-ENV HELM_VERSION v2.12.2
+ENV HELM_VERSION v2.13.1
 ENV HELM_DIFF_VERSION v2.11.0+3
 ENV HELM_SECRET_VERSION v1.3.1
-ENV AWS_IAM_AUTH_VERSION 1.11.5/2018-12-06
+ENV HELMFILE_VERSION v0.54.0
+ENV AWS_IAM_AUTH_VERSION 1.12.7/2019-03-27
 ENV VAULT_VERSION 0.10.1
```

It will be great to update vault as well.
```suggestion
ENV VAULT_VERSION 1.0.1
```